server/src/subscriber/subscriber.service.ts

```diff
@@ -145,6 +145,66 @@ export class SubscriberService {
     }
   }
 
+  /**
+   * Fetch the user email
+   * @param {string} id - The user's zap_production_id or zap_staging_id.
+   * @returns {object}
+   */
+  async getUserById(id: string) {
+    const query = `${this.sendgridEnvironmentIdVariable} LIKE '${id}'`
+    const request = {
+      url: `/v3/marketing/contacts/search`,
+      method: <HttpMethod>'POST',
+      body: { query }
+    }
+
+    // https://www.twilio.com/docs/sendgrid/api-reference/contacts/search-contacts
+    // https://www.twilio.com/docs/sendgrid/for-developers/sending-email/segmentation-query-language
+    try {
+      const users = await this.client.request(request);
+      if (users[0].body["contact_count"] === 0) {
+        return { isError: true, code: 404, message: "No users found." };
+      }
+      const email = users[0].body["result"][0].email;
+
+      return { isError: false, code: users[0].statusCode, "email": email };
+    } catch (error) {
+      return { isError: true, ...error };
+    }
+  };
+
+
+  /**
+   * Update user subscription.
+   * @param {string} environment - The environment variable
+   * @param {string} email - The user's email
+   * @param {object} custom_fields - The users custom lists and preferences
+   * @returns {object}
+   */
+  async update(environment: string, email: string, custom_fields: object) {
+    var updated_custom_fields = Object.entries(custom_fields).reduce((acc, curr) => ({ ...acc, [`zap_${environment}_${curr[0]}`]: curr[1] }), {})
```

Comment:
Can we stick with `let` or `const` and camelCased variable/parameter names?

Reply:
Do you want me to update to `let` and camelCase for the `create` in here as well ?